### PR TITLE
Add build

### PR DESCRIPTION
**Pull request by hiimdoublej-swag**
- [x] `libxslt`.
- [x] Add `arm64` for local testing.

### CODE REVIEW

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -13,7 +13,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: []
+        version:
+        - 1.0.0
```
Comment (allanlei) [inline .github/workflows/docker.yaml:17]: Is this the correct version?

**Review by allanlei — CHANGES_REQUESTED** [review]

File: .github/workflows/docker.yaml (hunk 1 above)
Comment (hiimdoublej-swag) [reply .github/workflows/docker.yaml]: Updated

File: Dockerfile [hunk 2]
```diff
@@ -1,21 +1,35 @@
-ARG         base=alpine
+ARG         base=alpine:3.17
```
Comment (allanlei) [inline Dockerfile:1]:
```suggestion
ARG         base=alpine:3.18
```

File: .github/workflows/docker.yaml [hunk 3]
```diff
@@ -56,13 +62,14 @@ jobs:
         flavor: |-
           latest=${{ !!matrix.latest }}
         tags: |-
-          type=semver,pattern={{major}}.{{minor}},value=${{ matrix.version }}
+          type=semver,pattern={{major}}.{{minor}}.{{patch}},value=${{ matrix.version }}
```
Comment (allanlei) [inline .github/workflows/docker.yaml:65]:
```suggestion
          type=semver,pattern={{major}}.{{minor}},value=${{ matrix.version }}
```

**Review by allanlei — APPROVED** [review]

**Pull request merged**